core/kazoo_number_manager/src/knm_port_request.erl

```diff
@@ -466,7 +466,10 @@ is_in_account_and_in_service(AccountId, PN) ->
 %%------------------------------------------------------------------------------
 -spec clear_numbers_from_port(kz_json:object()) -> {ok, kz_json:object()}.
 clear_numbers_from_port(PortReq) ->
-    Cleared = kz_json:set_value(?NUMBERS_KEY, kz_json:new(), PortReq),
+    Update = [{?NUMBERS_KEY, kz_json:new()}
```

Comment:
@icehess would want to be able to find ports by number in either list...

IE: when attempting to find the port request for 555 111 2222 should check in both ?NUMBERS_KEY and "ported_numbers".

Reply:
`all_port_in_numbers` and `listing_by_number` views are updated to emit number in both `numbers` and `ported_numbers` key. Both is only used by cb_port_requests to list port requests.